tests/unit/mixins/test_mixin_methods.py

```diff
@@ -205,6 +205,25 @@ class M(ListMixin, FakeManager):
     assert responses.assert_call_count(url, 2) is True
 
 
+@responses.activate
+def test_list_mixin_with_attributes(gl):
+    class M(ListMixin, FakeManager):
+        _types = {"my_array": gl_types.ArrayAttribute}
+
+    url = "http://localhost/api/v4/tests?my_array%5B%5D=1&my_array%5B%5D=2&my_array%5B%5D=3"
```

Comment:
I checked locally and the query params are not required here. They are checked by the `query_param_matcher` already. E.g. test it by changing the matcher to `my_error[]`:

```log
E           requests.exceptions.ConnectionError: Connection refused by Responses - the call doesn't match any registered mock.
E           
E           Request: 
E           - GET http://localhost/api/v4/tests?my_array%5B%5D=1&my_array%5B%5D=2&my_array%5B%5D=3
E           
E           Available matches:
E           - GET http://localhost/api/v4/tests Parameters do not match. {my_array[]: [1, 2, 3]} doesn't match {my_error[]: [1, 2, 3]}
```


```suggestion
    url = "http://localhost/api/v4/tests"
```

Reply:
Nice. Thanks!